docs-md/tutorials/embedded-connect.md

```diff
@@ -0,0 +1,409 @@
+---
+layout: page
+title: ksqlDB with Embedded Connect
+tagline: Run Kafka Connect embedded within ksqlDB
+description: Learn how to use ksqlDB with embedded Connect to integrate with external data sources and sinks
+keywords: ksqlDB, connect, PostgreSQL, jdbc
+---
+
+Overview
+==============
+
+This tutorial will demonstrate how to integrate ksqlDB with an external data
+source to power a simple ride sharing app. Our external source will be a
+PostgreSQL database containing relatively static data describing each driver’s
+vehicle. By combining this human-friendly static data with a continuous stream
+of computer-friendly driver and rider location events, we derive an enriched
+output stream that the ride sharing app may use to facilitate a rendezvous in
+real time.
+
+1. Get ksqlDB
+--------------
+
+Since ksqlDB runs natively on {{ site.aktm }}, you need a running {{ site.ak }}
+installation that ksqlDB is configured to use. The following docker-compose
+files run everything for you via Docker, including ksqlDB running
+[Kafka Connect](https://docs.confluent.io/current/connect/index.html) in
+embedded mode. Embedded Connect enables you to leverage the power of
+{{ site.kconnect }} without having to manage a separate {{ site.kconnect }}
+cluster, because ksqlDB manages one for you. Also, this tutorial use PostgreSQL
+as an external datastore to integrate with ksqlDB.
+
+In an empty local working directory, copy and paste the following
+`docker-compose` content into a file named `docker-compose.yml`. You will
+create and add a number of other files to this directory during this tutorial.
+
+```yaml
+---
+version: '2'
+
+services:
+  zookeeper:
+    image: confluentinc/cp-zookeeper:5.3.1
```

Comment:
5.3.2 is now the latest for all CP components (with 5.4.0 coming soon)

Reply:
updated